samri/plotting/_examples/test_stat3D.py

```diff
@@ -0,0 +1,13 @@
+def test_stat3D():
+	import samri.plotting.maps as maps
+
+	stat_map = "/home/gentoo/src/SAMRI/samri/plotting/allneg.nii.gz"
```

Comment:
Let's use packaged files for testing, i.e. those from the mouse-brain-atlases package. True, you do not have a “real” feture map in this package, but it doesn't matter, just use any other NIfTI file in the same space.